scale/product/models.py

```diff
@@ -606,6 +606,11 @@ def upload_files(self, file_entries, input_file_ids, job_exe, workspace):
             product.source_collection = entry.source_collection
             product.source_task = entry.source_task
 
+            # Update product model with details derived from the job_type
+            product.job_name = job_exe.job_type.name
+            product.job_version = job_exe.job_type.get_job_version()
+            product.package_version = job_exe.job_type.get_package_version()
+
```

Comment:
Sorry for the confusion, but the `job_name`, `job_version`, `package_version`, and `url` are supposed to go in the product's meta-data, not new fields on the product model. They should be added to the JSON meta-data that is saved in the product's `meta-data` field.